### PR TITLE
Camera patch

### PR DESCRIPTION
**Pull request by deven96**

### CODE REVIEW

File: ImageQ/processor/base.py [hunk 1]
```diff
@@ -1,54 +1,168 @@
...
 
 
 __all__ = [
+    'BaseSearch',
     'BasePredictor',
     'BaseHandler',
 ]
 
 
+class BaseSearch(object):
```
Comment (MeNsaaH) [inline ImageQ/processor/base.py]: Was there a reason for this, didn't the `search-engine-parser` package work?

File: README.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ The prediction API code can be found at the [repo](https://github.com/deven96/Im
 
 The `master` branch of the repo is linked to automatically [deployed](https://bisoncorps-imageq.herokuapp.com)
 ![Homepage](docs/assets/homepage.png)
- and it sends requests to the deployed [prediction API](https://imageqapi.appspot.com/)
+and it sends requests to the deployed [prediction API](https://imageqapi.appspot.com/)
```
Comment (MeNsaaH) [inline README.md]: We need to add a `pylint` and `autopep8` configuration that handles things like white space and setup a commit hook that auto formats the code before pushing. 
I'll create an issue and self assign it

**Comment by deven96** [comment]
Yeah it has one very funny issue I haven't figured out .... using it in the
imageQ is  okay o... but once I try using the package it doesn't work...
maybe you could help debug it

**Comment by deven96** [comment]
I've been thinking of autopep8 as well... it's really necessary... maybe
add it as a sort of commit hook

**Comment by deven96** [comment]
Also we can put issues with the help wanted label so Github can suggest the
repo for people to contribute

On 3 Feb 2019 22:00, "MeNsaaH" <notifications@github.com> wrote:

*@MeNsaaH* commented on this pull request.

We need to add a pylint and autopep8 configuration that handles things like
white space and setup a commit hook that auto formats the code before
pushing.
What's up with the search-engine-parser?
------------------------------

In ImageQ/processor/base.py
<https://github.com/bisoncorps/imageQ/pull/6#discussion_r253319002>:

>      'BasePredictor',
     'BaseHandler',
 ]


+class BaseSearch(object):

Was there a reason for this, didn't the search-engine-parser package work?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/bisoncorps/imageQ/pull/6#pullrequestreview-199404762>,
or mute the thread

**Comment by MeNsaaH** [comment]
Alright then. I will look into it and the `search-engine-parser`

**Pull request merged**